about.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>anderspace</title>
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="styles/about.css">
+    <link rel=icon href="purple_heart.png">
+  </head>
+
+  <body>
+    <div id="header-style">
+      <div class="wrapper">
+        <header class="clearfix">
+          <h3>(a bit) About me</h3>
+          <nav>
+            <ul>
+              <li class="inline-block">
+                <a href="index.html">
+                  Home
+                </a>
+              </li>
+              <li class="inline-block">
+                <a href="portfolio.html">
+                  Portfolio
+                </a>
+              </li>
+              <li class="inline-block">
+                <a href="code_journal.html">
+                  Code Journal
+                </a>
+              </li>
+              <li class="inline-block">
+                <a id="highlight" href="about.html">
+                  About
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </header>
+      </div>
+    </div>
+
+    <main>
+      <section class="main-background-image">
+        <div class="wrapper">
+          <p class="margin-zero">
+            Thanks for looking around!
+          </p>
+          <section class="main-content">
+            <article>
+              <p id="names" class="margin-zero">
+                I am:
+              </p>
+              <p class="margin-zero">
+                <span>anders</span>
+                <span>Andrea</span>
+                <span>娃娃</span></p>
+              <p class="margin-zero">
+                Taiwanese-Canadian
+              </p>
+              <p class="margin-zero"> — </p>
+              <p id="hobbies" class="margin-zero">
+                I like:
+              </p>
+              <p class="margin-zero">
+                <span>dance</span>
+                <span>birds</span>
+                <span>astromony</span>
+              </p>
+              <p class="margin-zero"> — </p>
+              <p id="last-line-about" class="inline-block margin-zero">
+                And I live over there
+              </p>
+              <img src="images/white_arrow_2.png" alt="arrow">
+            </article>
+          </section>
+        </div>
+      </section>
+
+      <footer>
```

Comment:
I think usually the `footer` tag is placed outside of the `main` tag, rather than nested within it as we have here.